src/datahike/migrate.clj

```diff
@@ -1,15 +1,20 @@
 (ns ^:no-doc datahike.migrate
   (:require [datahike.api :as api]
+            [datahike.constants :as c]
             [datahike.datom :as d]
             [datahike.db :as db]
             [clj-cbor.core :as cbor]))
 
 (defn export-db
   "Export the database in a flat-file of datoms at path."
-  [db path]
-  (cbor/spit-all path (map seq (sort-by
-                                (juxt d/datom-tx :e)
-                                (api/datoms (if (db/-keep-history? db) (api/history db) db) :eavt)))))
+  [conn path]
+  (let [db @conn
+        cfg (:config db)]
+    (cbor/spit-all path (cond->> (sort-by
+                                  (juxt d/datom-tx :e)
+                                  (api/datoms (if (:keep-history? cfg) (api/history db) db) :eavt))
+                          (:attribute-refs? cfg) (remove #(= (d/datom-tx %) c/tx0))
```

Comment:
What happens if the system entities are changed between export and import datahike versions? For example when the order changes or there are more or less system entities than before?

Reply:
I assumed any changes to system entities would only be additive. I guess that isn't a sufficiently realistic or safe assumption? Also, as I understand, the migration code here isn't a permanent solution—a role Wanderung is supposed to fulfil at some point. Please let me know.

Reply:
I guess, if it is only a temporary solution, it should be ok for now, but maybe put at least a note in there about the assumption. 
I was of the impression, the system entities are in the database for exactly that reason - so the case of invalid references would never occur. But I wouldn't be surprised if none of our functions support importing the schema and I'd say that would be my mistake, sorry. I will open an issue or discussion about it to not lose track of the subject.